BSP/STM32F413/Src/BSP_PWM.c

```diff
@@ -0,0 +1,170 @@
+ /* Copyright (c) 2018-2022 UT Longhorn Racing Solar */
+
+#include "BSP_PWM.h"
+
+GPIO_InitTypeDef GPIO_INIT_STRUCT; //struct used to initialize pins
+TIM_OCInitTypeDef TIMER_OC_STRUCT; //struct used to configure output compare for timers
+TIM_TimeBaseInitTypeDef TIMER_INIT_STRUCT; //struct used to initialize PWM timers
+
+
+/**
+ * @brief   Sets up contactor and fan pin timers for outputting PWM signals
+ * @note    GPIO_B0 = contactor
```

Comment:
doesn't PWM control more than just the fans and fan contactor? There's should be at least one other pin for the other two contactors.

Reply:
That pin is shared by 2 contactors. the other 4 are just for fans.